model/app.model.js

```diff
@@ -67,41 +67,66 @@ const selectAllCommentsForArticle = (id) => {
 
 
 const insertComment = (id, username, body) => {
-    if (Number.isNaN(parseInt(id))){
-        return Promise.reject({
-            status: 400,
-            msg: "Invalid article id"
-            })
-    }
-    if (isEmpty(username) || isEmpty(body)){
+    if (isEmpty(username)){
         return Promise.reject({
         status: 400,
-        msg: "Missing comment body and or author name"
+        msg: "Missing author name in the comment"
         })
     }
+    if (isEmpty(body)){
+        return Promise.reject({
+            status: 400,
+            msg: "Missing comment body"
+            })
+    }
     return db.query(`INSERT INTO comments(author, body, article_id)
     VALUES ($1, $2, $3)
     RETURNING *;`, [username, body, id])
     .then((result) => {
+            return result.rows[0] 
+        }    
+    )
+}
+
+function isEmpty(value) {
+    return (value == null || (typeof value === "string" && value.trim().length === 0));
+}
+
+const addNewVotes = (article_id, inc_votes) => {
+    if (isEmpty(inc_votes)){
```

Comment:
If inc votes is missing, ideally we just want to return the article object unchanged, rather than throwing an error here :)